core/src/main/java/io/parsingdata/metal/expression/value/reference/CurrentOffset.java

```diff
@@ -25,7 +25,8 @@
 import io.parsingdata.metal.expression.value.ValueExpression;
 
 /**
- * Retrieve the current environment offset.
+ * A {@link ValueExpression} that represents the current offset in the parse
+ * state.
```

Comment:
Well, this is because I'm wondering whether we should explicitly discuss the names and shape of the `ParseGraph` and `Environment` APIs, or maybe let it be more conceptual at this level (and the documentation more durable).